src/integrated/soil_canopy_model.jl

```diff
@@ -199,6 +204,11 @@ interaction_domain_names(m::SoilCanopyModel) = (
     :surface,
     :surface,
     :surface,
+    :surface,
+    :surface,
+    :surface,
+    :surface,
+    :surface,
```

Comment:
What's going here?

Reply:
In the land model, we have variables that live on different parts of the domain. These methods (along with ones defining variable symbol names and variable types) help us create the prognostic state and auxiliary state in an automated way.